lib/python/rosie/vc.py

```diff
@@ -354,11 +394,22 @@ def generate_info_config(self, from_id=None, prefix=None):
         else:
             info_config.set(["project"], "")
         if from_title:
-            info_config.set(["title"], "Copy of %s: %s" % 
-                            (from_id.to_string_with_version(), from_title))
+            info_config.set(["title"], from_title)
         else:
             info_config.set(["title"], "")
 
+        # Determine description and subproject
+        if from_description:
```

Comment:
Does this actually copy project-specific options across, or does it just copy description, title, project, and sub-project?